server/services/web.ts

```diff
@@ -42,6 +43,14 @@ if (env.GOOGLE_ANALYTICS_ID) {
   scriptSrc.push("www.google-analytics.com");
 }
 
+if (environment.UMAMI_SCRIPT_DOMAIN) {
```

Comment:
Please remove changes in this file, csp is injected automatically for analytics integrations